processing/proc_ssd.m

```diff
@@ -94,6 +94,7 @@
 
 %% rename channel labels and save old channel labels
 dat.origClab= dat.clab;
-for k=1:length(dat.clab)
+dat.clab={};
```

Comment:
That looks better! Final suggestion: how about 
dat.clab = cell(1,size(dat.x,2)); 
?